browser/components/SnippetTab.js

```diff
@@ -85,6 +85,19 @@ class SnippetTab extends React.Component {
     })
   }
 
+  handleDragStart (e) {
+    e.dataTransfer.dropEffect = 'move'
+    this.props.onDragStart(e)
+  }
+
+  handleDragEnd (e) {
+    this.props.onDragEnd(e)
```

Comment:
There are no props `onDragEnd`.
Could you delete this function?
Or did you forget to implement?

Reply:
I started with using onDragStart and onDragEnd, but decided that I needed the onDrop for this job. I removed the handler.